rllib/algorithms/algorithm.py

```diff
@@ -2332,7 +2332,7 @@ def export_policy_checkpoint(
                 will raise a KeyError.
 
         Raises:
-            KeyError if `policy_id` cannot be found in this Algorithm.
+            KeyError: if `policy_id` cannot be found in this Algorithm.
```

Comment:
why is this change also required?

Reply:
This is the googledoc syntax for exceptions - without it the reference fails to resolve.